File: metrics/test/data/test_cognitive_complexity/test_combination/data_cognitive_complexity.cs
public class TestClass
{
    public TestClass() 
    { 
        if (true)
        {
            if (false)
            {
                break;
            }
        }
        while (true)
        {
            continue;
        }
        try 
        {
            Console.WriteLine();
        }
        catch (Exception e)
        {
            Console.WriteLine();
        }
    }
    public void method1() 
    {
        if (true) 
        {
            if (false)
            {
                method1();
            }
        }
        while (true)
        {
            try
            {
                Console.WriteLine("E");
            }
            catch (Exception e)
            {
                continue;
            }
        }
    }

}

File: metrics/test/data/test_cognitive_complexity/test_combination/data_cognitive_complexity.java
public class TestClass {
    public TestClass() {
        if (a) { //1
            if (b) { //2
                break; //3
            }
        }
        for (Integer i = 0; i < 10; i++) //4
        {
            continue; //5
        }
        break;
        while(true) { //6
            System.out.println("Cool");
        }
    }
    public void method1() {
        if (a) { //1
            if (b) { //2
                method1(); //3
            }
        }
        for (Integer i = 0; i < 10; i++) //4
        {
            method1(); //5
        }
        while(true) { //6
            System.out.println("Cool");
        }
            
    }
}